frameworks/Java/quarkus/benchmark_config.json

```
@@ -55,7 +41,7 @@
         "flavor": "None",
         "orm": "Micro",
         "platform": "JAX-RS",
-        "webserver": "None",
+        "webserver": "Undertow",
```

```suggestion
        "webserver": "None",
```
the pgclient implementation is using reactive routes from from vert.x-web. This should not be Undertow